app/models/user.rb

```diff
@@ -2,6 +2,8 @@
 class User < ActiveRecord::Base
   has_secure_password
 
+  has_many :polls, dependent: :destroy
+
   validates :username, presence: true
```

Comment:
So usernames are allowed to be not unique?
It might open doors for some nasty impersonation fraud cases, can't it?